NEWS.txt

```diff
@@ -1,3 +1,11 @@
+0.8.4 (2016-02-17):
+  - Remove unused `nevow.livepage` support code; fixes compatibility with Nevow
+    0.13.0.
+  - StringEndpointPort ports are now deletable from the commandline (with
```

Comment:
Nope, added them. Although this file is ".txt" so they don't really do anything; should rename it to ".rst" I guess?